src/js/util.js

```diff
@@ -28,4 +27,16 @@ export function makeId() {
   }
   return s4() + s4() + '-' + s4() + '-' + s4() + '-' +
     s4() + '-' + s4() + s4() + s4();
-}
\ No newline at end of file
+}
+
+// pick a random oject from list
+export function choice(list) {
+  return list[Math.floor(Math.random() * list.length)]
+}
+
+// these gets repeated in SCSS, smelly
```

Comment:
No need for these to be defined here. Query the DOM where you actually need these things.